Casks/e/epicenter-whispering.rb

```diff
@@ -0,0 +1,31 @@
+cask "epicenter-whispering" do
+  arch arm: "aarch64", intel: "x64"
+
+  version "7.4.0"
+  sha256 arm:   "a7e301a371ce45d8224aea3079bfe9c9ae568f413e72d36d1fe6a2e46c6cf494",
+         intel: "cb053aa69fa9fb315f98233cb54831df3b9e527230dabde6babc7e6af39bd303"
+
+  url "https://github.com/epicenter-os/epicenter/releases/download/v#{version}/Whispering_#{version}_#{arch}.dmg",
+      verified: "github.com/epicenter-os/epicenter/"
+  name "Epicenter Whispering"
+  desc "Fast, minimal transcription that works with local and cloud models"
```

Comment:
```suggestion
  desc "Audio transcription that works with local and cloud models"
```

We avoid using marketing/empty descriptors in desc stanzas.